src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,75 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.26.0.dev2 (02/27/2020)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Upgrade to Pex 2.1.5. (#9187)
+  `PR #9187 <https://github.com/pantsbuild/pants/pull/9187>`_
+
+* Add back `ignore` for `--files-not-found-behavior` (#9175)
+  `PR #9175 <https://github.com/pantsbuild/pants/pull/9175>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Add support for reading Python lock files in V2 (#9163)
+  `PR #9163 <https://github.com/pantsbuild/pants/pull/9163>`_
```

Comment:
Probably worth calling attention to in the email? This is one of the most requested features Pants has.

Reply:
We should probably disclose in the email that this is for V2. Then we could put a brief pitch "Not using the V2 Python experience yet? We'd love to help you try it out. Head over to Slack for help upgrading."